content/news/2024-05-28-CECAM-conference/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Develop and engage: Sustainable software tools for large-scale facilities using the galaxy platform"
+tease: "Talk on Galaxy tools for muon experiments and catalysis research at the CECAM conference"
+hide-tease: false
+authors: Leandro Liborio, Patrick Austin, Subindev Devadasan, Abraham Nieva de la Hidalga, Alejandra Gonzalez-Beltran
+date: "2024-05-28"
+days: 3
```

Comment:
```suggestion
date: "2024-05-28"
days: 3
```

Isn't that only for events?

Reply:
Meaning this folder: https://github.com/galaxyproject/galaxy-hub/tree/master/content/events